.cirrus.yml

```diff
@@ -28,11 +28,23 @@ task:
     - cd ../Pcap++Test && Bin/Pcap++Test -n
 
 task:
-  name: Regression Tests Linux
+  name: Regression Tests Linux (Address Sanitizer)
   container:
     image: seladb/ubuntu1804:latest
   configure_script:
-    - ./configure-fuzzing.sh --default
+    - ./configure-fuzzing.sh --sanitizer address
+    - ldconfig
+  build_script:
+    - make fuzzers
+  test_script:
+    - cd Tests/Fuzzers/RegressionTests && ./run_tests.sh
+
+task:
+  name: Regression Tests Linux (Undefined Behavior Sanitizer)
+  container:
+    image: seladb/ubuntu1804:latest
+  configure_script:
+    - ./configure-fuzzing.sh --sanitizer undefined
```

Comment:
the `undefined` option also runs it with address sanitizer. Why do we need 2 tests with the same configuration

Reply:
What do you mean that it also runs it with ASan? The `--sanitizer undefined` option of the `configure-fuzzing.sh` script only builds PcapPlusPlus and the fuzzers with the Undefined Behavior Sanitizer, but not with Address Sanitizer. Thus the idea of having 2 separate tests (also OSSFuzz runs each sanitizer on a different fuzzer build).

Reply:
oh you're right, my bad. What does `undefined` sanitizer mean btw?

Reply:
No problem! That's the name for the [Undefined Behavior Sanitizer](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html)

Reply:
got it thanks!